ErxiaHQ/govote-api
# co-tests

#### What does this PR do?
- This PR sets up and adds tests to the go-vote project

#### Description of Task to be completed?
- Add tests to the go-vote project

#### How should this be manually tested?
- In the app root directory, run `npm-test` to run the test and see the collected coverage

#### What is the relevant issue for this PR?
- [#1](https://github.com/ErxiaHQ/govote-api/issues/1)

#### Any background context you want to add?
- N/A

#### Screenshots
![image](https://user-images.githubusercontent.com/8125356/38768818-22a6e45a-3ff1-11e8-8090-499fc2a94848.png)
